satpy/readers/seviri_l1b_native.py

```diff
@@ -343,9 +418,8 @@ def get_area_extent(self, dsid):
 
         return area_extent
 
-    def get_dataset(self, dsid, info,
-                    xslice=slice(None), yslice=slice(None)):
-
+    def get_dataset(self, dsid, info):
```

Comment:
For clarity, please rename `dsid` to `dataset_id` and `info` to `dataset_info`. Apply the same elsewhere in the reader and tests, if/when necessary.